grouper/fe/routes.py

```diff
@@ -25,6 +25,7 @@
     ),
     (r"/search", handlers.Search),
     (r"/users", handlers.UsersView),
+    (r"/users/public-keys", handlers.UsersPublicKey),
```

Comment:
These are specifically SSH keys, right? Will this endpoint return, e.g., X.509 public keys if Grouper grows support for that in the future?